data/templates/accel-ppp/pppoe.config.tmpl

```diff
@@ -100,8 +100,8 @@ ac-name={{ access_concentrator }}
 {%   for iface in interface %}
 interface={{ iface }}
 {%     if interface[iface].vlan_id is defined or interface[iface].vlan_range is defined %}
-vlan-mon={{ iface }},{{ interface[iface].vlan_id | join(',') }},{{ interface[iface].vlan_range | join(',') }}
-interface=re:{{ interface.name }}\.\d+
+vlan-mon={{ iface }},{{ interface[iface].vlan_id | join(',') if interface[iface].vlan_id is defined }}{{ interface[iface].vlan_range | join(',') if interface[iface].vlan_range is defined }}
```

Comment:
Could we use the opportunity when changing this template to also "refactor" it a bit?

Writing
 ```
{% if interface is defined and interface is not none %}
{%   for iface, iface_config in interface.items %}
{%     if iface_config.vlan_id is defined or iface_config.vlan_range is defined %}
...
```

The `.item()` statement will infact map the dictionary key to `iface` and the corresponding dictionary value to `iface_config`. By this we can have cleaner code and simply use `iface_config` over `interface[iface]`

Reply:
Done